### PR TITLE
Add GitHub icon and Code link to site footer

### PR DESCRIPTION
**Pull request by critzo**
This PR adds a Github logo and Code link to the M-Lab org github in the site footer per #306 
Previewable on my fork: https://critzo.github.io/m-lab.github.io/
@georgiamoon PTAL?

---
This change is [Reviewable](https://reviewable.io/reviews/m-lab/m-lab.github.io/313)

### CODE REVIEW

File: _data/header-footer-content.yml [hunk 1]
```diff
@@ -85,6 +85,10 @@ footer:
   subscribe:
     items:
     - 
+      link: "https://github.com/m-lab"
```
Comment (georgiamoon) [inline _data/header-footer-content.yml]: The links seem to be concatenating to the site URL, so on the preview it looks like these won't work.

File: _data/header-footer-content.yml [hunk 2]
```diff
@@ -126,6 +130,9 @@ footer:
     column-three:
       items:
       -
+        link: "https://github.com/m-lab"
```
Comment (georgiamoon) [inline _data/header-footer-content.yml]: Same issue here

**Comment by georgiamoon** [comment]
Visually looks good, but there's an issue with the links on the built page.

**Comment by critzo** [comment]
This is only an issue on the forked site. URLs will resolve correctly on the production site.

**Comment by georgiamoon** [comment]
OK -- Well I'll guess we'll confirm quickly! LGTM

**Pull request merged**